src/gfx_device/gfx_device.rs

```diff
@@ -59,7 +59,7 @@ impl GfxDevice {
         // the scene.
         for (rend, entity) in (&renderables, &entities).iter() {
             let global_trans = match global_transforms.get(entity) {
-                Some(gt) => gt.clone(),
+                Some(gt) => *gt,
```

Comment:
I think that if you change `Some(gt)` to `Some(ref gt)`, you can omit the deref asterisk.